### DISCUSSION TITLE
Picture Selection Task

### DISCUSSION THREAD

**Discussion by hblam** [jspsych/jsPsych#3181] [Q&A] [answered]
Hi! With this being my first time using jspsych, I am having trouble getting images to load side by side for a picture selection task. I would like two pictures to show up (1a and 1b) and for the participants to select which picture best matches the sentence described (which will derive from a self-paced reading component of the experiment).

Right now, the file structure has the experiment alongside a folder 'img' that contains the .png files. When I try to run the experiment, it fails to show any pictures. Below is what I have so far.

```
<!DOCTYPE html>
<html>
<head>
  <script src="../dist/jspsych.js"></script>
  <script src="../dist/plugin-html-keyboard-response.js"></script>
  <script src="../dist/plugin-preload.js"></script>
  <link rel="stylesheet" href="../dist/jspsych.css">
</head>
<body></body>
<script>

  var jsPsych = initJsPsych({
    on_finish: function() {
      jsPsych.data.displayData();
    }
  });

  var preload = {
      type: jsPsychPreload,
      auto_preload: true
    };

  var trial_1 = {
    type: jsPsychHtmlKeyboardResponse,
    stimulus: '<img src=img/1a.png /><img src=img/1b.png />',
    choices: ['1', '0'],
    prompt: '<p>Which picture matches the sentences (1 for left or 0 for right)</p>'
  };

  jsPsych.run([trial_1]);

</script>
</html>
```

**Comment by jodeleeuw** [comment, marked as answer]
Hi @hblam

There are two things to change here:

1. Your `stimulus` parameter for `trial_1` is missing `"` around the file names. It should be `'<img src="img/1a.png" /><img src="img/1b.png" />'`
2. The `auto_preload` method for the preload trial won't work in this context because the images are added as HTML instead of through an image-based plugin. Instead use:

```js
var preload = {
      type: jsPsychPreload,
      images: ['img/1a.png', 'img/1b.png']
 };
```

**Reply by hblam** [reply]
Thank you for your reply! It worked!!!
